### DISCUSSION TITLE
Database migration on docker or production environment

### DISCUSSION THREAD

**Discussion by cephaspad** [abpframework/abp#16282] [Q&A] [answered]
Hello, 

Anyone, please, help me to migrate database on production or docker environment.

I use **Abp Framework 7.1**

I published the **DbMigrator** project with image: **aspnet:7.0.**

When I run **dotnet run [ProjectName].DbMigrator.dll.**

The error show: **No project was found. Change the current working directory or use the --project option.**

**Comment by maliming** [comment, marked as answer]
>  No project was found. 

Check the DLL files of current directory.
